client/src/app/shared/components/Notifications.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import {
+  Alert,
+  AlertActionCloseButton,
+  AlertGroup,
+} from "@patternfly/react-core";
+import { NotificationsContext } from "../notifications-context";
+
+export type notificationType =
+  | "success"
+  | "info"
+  | "warning"
+  | "danger"
+  | "default";
+
+export interface INotification {
```

Comment:
It seems like maybe these types should be defined in the notifications-context file rather than the Notifications component file, since the context provider is where notifications are stored and passed down? not sure if that matters.